base_user_role/views/user.xml

```diff
@@ -39,4 +39,14 @@
             </field>
         </field>
     </record>
+    <record id="view_res_users_tree_inherit" model="ir.ui.view">
+        <field name="name">res.users.tree.inherit</field>
```

Comment:
not blocking point : FYI this field is useless. (and is autogenerated, if not set).